Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
It won't be an exact fit I suspect but the mechanism itself would likely be similar to [mpd](https://github.com/Homebrew/homebrew-core/blob/55183eb837f5535ff1b57051dbd0f2bd282ea3e0/Formula/mpd.rb#L127-L139)'s test.